build-support/bin/release.sh

```diff
@@ -492,6 +436,7 @@ function fetch_and_check_prebuilt_wheels() {
   fetch_prebuilt_wheels "${check_dir}"
 
   local missing=()
+  RELEASE_PACKAGES=($(run_local_pants -q run src/python/pants/releases:packages -- list | grep '.' | awk '{print $1}'))
```

Comment:
Maybe extract a small bash funtion for this - it's used 2x and semi-non-straightforward.